### ISSUE TITLE
Remove logo alt text in bar

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine#795]
From VT's accessibility audit:

> The LSV_bug.png image has alt=”Home - Assembly Line” . Since this image is contained within a
link along with other text, it should have alt=”” since (a) it adds no information and (b) the link is
adequately represented by the text. 3-Moderate

This is my fault, I added some default alt text instead of doing `alt=""`, but it's an easy fix.
